docs/lib/analytics/fragments/android/getting-started/20_installLib.md

```diff
@@ -0,0 +1,20 @@
+* `mavenCentral()` as a repository
+
+```groovy
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.5.0'
+    }
+}
+```
```

Comment:
Sorry - I should have been more clear - all of these lines are unnecessary.

Reply:
Sorry again that I don't familiar with Android. Removed